routes/products.js

```diff
@@ -0,0 +1,112 @@
+const express = require('express');
+const routerProducts = express.Router();
+const knex = require('../database');
+
+routerProducts.route('/').get((req, res) => {
+  knex
+    .raw('SELECT * FROM products')
+    .then((result) => {
+      res.send(result.rows);
+    })
+    .catch((err) => {
+      res.send(500, '{ "message": "Database erorr" }');
+    });
+});
+
+routerProducts
+  .route('/:id')
+  .get((req, res) => {
+    const id = req.url.slice(1);
+    knex
+      .raw('SELECT * FROM products WHERE id = ?', [id])
+      .then((result) => {
+        if (result.rows.length === 0) {
+          return res.send('{ "message": "Product not found" }');
+        }
+        res.send(result.rows);
```

Comment:
In this case, it's probably best to send back the single object in the rows rather than the array of one item.